src/qibolab/qubits.py

```diff
@@ -38,14 +38,25 @@ class Qubit:
     name: QubitId
 
     bare_resonator_frequency: int = 0
-    readout_frequency: int = 0  # this is the dressed frequency
+    readout_frequency: int = 0
+    """ Readout dressed frequency"""
     drive_frequency: int = 0
     anharmonicity: int = 0
-    Ec: int = 0
-    Ej: int = 0
-    g: int = 0
+    sweetspot: float = 0.0
+    flux_to_bias: float = 0.0
+    asymmetry: float = 0.0
+    brf: float = 0.0
+    """Bare resonator frequency obtained during resonator flux spectrosocopy"""
```

Comment:
Is this different than the existing `bare_resonator_frequency` because you are at the sweetspot?

Reply:
@stavros11 you are right. I have modified a little bit the docstring to be more precise